README.rst

```diff
@@ -121,6 +121,18 @@ Furthermore, there are several edX-specific environment variables that enable in
      - Enables support for opting in marketing emails that helps us getting user consent for sending marketing emails.
      - ``true`` | ``''`` (empty strings are falsy)
 
+   * - ``APP_ID``
+     - Enables support for runtime configurations that will be available in getConfig method specific to MFE
+     - ``true`` | ``''`` (empty strings are falsy)
+
+   * - ``MFE_CONFIG_API_URL``
+     - Enables support for runtime configurations that will be available for common fields
+     - ``true`` | ``''`` (empty strings are falsy) 
```

Comment:
These two are not edX specific but generic settings that can be used by both edX and open community.

Reply:
I have moved the above mentioned configs to general settings.

Reply:
I have moved the above mentioned configs to general settings.